tools/packaging/kata-deploy/Dockerfile

```diff
@@ -2,16 +2,15 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM centos/systemd
+FROM jrei/systemd-centos
```

Comment:
What if we use  centos base image and run this: https://github.com/CentOS/CentOS-Dockerfiles/blob/master/systemd/centos7/Dockerfile

Reply:
I'm experimenting with a few things for now, but CentOS images still have the a lot of vulnerabilities (the base CentOS image, this one ...).

I'd prefer going with the fedora-minimal one if I can make quay.io properly understand and scan the image, let's see.